transport/src/main/java/io/netty/channel/ChannelConfig.java

```diff
@@ -183,6 +183,17 @@
     ChannelConfig setRecvByteBufAllocator(RecvByteBufAllocator allocator);
 
     /**
+     * Returns {@code true} if {@link ChannelOption#AUTO_FLUSH} option is set to {@code true}.
+     */
+    boolean isAutoFlush();
+
+    /**
+     * Sets if explicit flushes are not required for this channel. If set to {@code true} every eventloop iteration
+     * will flush all pending writes. Explicit flushes will still be honored. The default value is {@code false}.
+     */
+    ChannelConfig setAutoFlush(boolean autoFlush);
```

Comment:
We can not add methods without moving to 4.2.x . Because of this we should not add the methods here yet but only use the new `ChannelOption` which is fine to add. Sounds good ? We can add them to `DefaultChannelConfig`tho.